docs/contributing-concept.md

```diff
@@ -0,0 +1,142 @@
+
+Identify a missing concept that you want to implement.
+Beware that the graph above is a moving target and may change as we find that prerequisites should differ.
+
+## 2. Define the learning goals
+
+It is important to know what a concept exercise aims to teach and what is out of scope so let's create our first file: `exercises/concept/<story-name>/.meta/design.md`.
+Since concept exercises are named after the story they use we don't know it yet, so we'll keep `story-name` for now and replace it later.
+The [`design.md` specification][design-spec] states that this document should define the overall goal of the exercise, its multiple learning objectives, the things that are out of scope, the associated concept it unlocks and the concepts prerequisites to be able to complete the exercise.
+You can use the [`design.md` of the boolean concept exercise][design-bool] as a template starting point.
+
+## 3. Write a concept introduction for the exercise
+
+Once the learning objectives are clear, it's time to write an introduction explaining enough of the concept for a student to then solve the exercise.
+Create a file `exercises/concept/<story-name>/.docs/introduction.md`.
+The detailed contents of this file are explained in [its specification][exercise-intro-spec].
+You can use the [`introduction.md` of the boolean concept exercise][intro-bool] as a template starting point.
+
+The information provided should introduce the concept just enough for the student to figure out the exercise solution themselves.
+Use proper technical terms to ease student online search but try to avoid links and to explain things concisely inline in the document.
+Introduce code syntax with code blocks.
+
+## 4. Find a story for the exercise
+
+Each concept exercise must be based on a story or a theme.
+We try to make them as inclusive as possible, knowing that Exercism is used worldwide.
+The first option is to reuse an existing story within the [list of stories recorded in the docs][stories].
+If you want to create a new story, don't hesitate to ping @exercism/leadership in your PR.
+You can now rename the `exercises/concept/<story-name>/` directory to match your story.
+
+## 5. Write the exercise instructions
+
+Once the story is found, we have to write precise instructions enumerating the tasks a student will have to perform.
+Both the story and the instructions are written in the `exercises/concept/<story-name>/.docs/instructions.md` document.
+As explained in [that document specification][instructions-spec], each task has its own level-2 markdown title (`## 1. Do X`) followed by a description and an example code usage.
+You can use the [`instructions.md` of the boolean concept exercise][instructions-bool] as a template starting point.
+
+In addition to `instructions.md`, we also need a `hints.md` file.
+It should contain the same level-2 titles (`## 1. Do X`) than the instructions file so that the right hints are given depending on the current task being worked on.
```

Comment:
```suggestion
It should contain the same level-2 titles (`## 1. Do X`) as the instructions file so that the right hints are given depending on the current task being worked on.
```